appdir.go

```diff
@@ -21,8 +23,14 @@ func Logs(app string) string {
 
 func InHomeDir(filename string) string {
 	usr, err := user.Current()
-	if err != nil {
-		panic(fmt.Errorf("Unable to determine user's home directory: %s", err))
+	if err == nil {
+		return filepath.Join(usr.HomeDir, filename)
+	}
+	// "user: Current not implemented on ..." will happen on Linux or Darwin
+	// when cross-compiled from other platforms.
+	homeDir, err2 := homedir.Dir()
```

Comment:
The reason why not use `go-homedir` exclusively is that sometimes, the user can override `HOME` environment, makes this method not that persistent.

Reply:
LGTM! Some questions:

- isn't `go-homedir` reliable enough that we can use it exclusively?

- if not, stylistic nitpick: wouldn't the idiomatic thing to do in golang be to first try and initialize `homeDir`, handling errors in (possibly nested) `if` blocks at the beginning, panic if nothing works, and otherwise let flow get to the final unindented `return` statement at the end?

Reply:
It must be my unnatural English. Regarding the style, yes, I actually did that first, and felt that the nested `if` clause is not that straightforward compared to the current form. Kinda a personal preference.